statsmodels/tsa2/sarimax/estimators/tests/results/results_burg.R

```diff
@@ -0,0 +1,26 @@
+library(itsmr)
```

Comment:
i wonder if we can smoke-test R scripts like this.  best-case would be to check their output is unchanged, but that would require more machinery.  I'm just thinking "this still runs correctly" tests

Reply:
(not necessarily for this PR, just thinking out loud)